src/sql/workbench/api/common/sqlExtHost.api.impl.ts

```diff
@@ -534,6 +534,7 @@ export function createAdsApiFactory(accessor: ServicesAccessor): IAdsExtensionAp
 
 			return {
 				accounts,
+				ButtonType: azdata.ButtonType,
```

Comment:
should be sqlExtHostTypes.ButtonType

Reply:
I see the **same** error whether I use `azdata` or `sqlExtHostTypes`.

Reply:
What error is that? Leila is correct - you need to use sqlExtHostTypes Button type here - azdata doesn't exist at runtime so the enum defined there isn't available.

Reply:
Implemented using: `sqlExtHostTypes`